app/index/template.hbs

```diff
@@ -6,12 +6,13 @@
           {{fa-icon "fa-plus-square"}}
         </button>
       {{/if}}
-      <input
+      {{input
         class='new-topic-input'
         type='text'
-        value={{newTopic}}
+        value=newTopic
         placeholder='Suggest a Topic'
-        oninput={{action "newTopicUpdated" value="target.value"}} />
+        key-press='newTopicUpdated'
+        value=target.value }}
```

Comment:
These two lines are actually unnecessary now.  The only reason we had `oninput` before was because you don't get 2-way data binding when using an html input so you have to catch the `oninput` and update the property manually.  Now that we are using the `input` helper 2-way data binding is free so we can just remove the line and the action handler in the actions hash on the route.